sklearn/metrics/tests/test_ranking.py

```diff
@@ -470,19 +507,24 @@ def test_precision_recall_curve_pos_label():
 def _test_precision_recall_curve(y_true, probas_pred):
     # Test Precision-Recall and aread under PR curve
     p, r, thresholds = precision_recall_curve(y_true, probas_pred)
-    precision_recall_auc = auc(r, p)
-    assert_array_almost_equal(precision_recall_auc, 0.85, 2)
+    precision_recall_auc = _average_precision_slow(y_true, probas_pred)
+    interpolated_average_precision = _interpolated_average_precision_slow(
+        y_true, probas_pred)
+    assert_array_almost_equal(precision_recall_auc, 0.859, 3)
```

Comment:
Agreed. Ping @ndingwall: we don't like to have hard-coded numbers in our test suites without being able to rederive them.